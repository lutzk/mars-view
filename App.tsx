import React from 'react';
import { SafeAreaView, StatusBar } from 'react-native';

import { NavigationContainer } from '@react-navigation/native';

import { MainStack } from './src/navigation/mainStack';

function App(): JSX.Element {
  return (
    <NavigationContainer>
      <SafeAreaView style={{ flex: 1 }}>
        <StatusBar
        // barStyle={isDarkMode ? 'light-content' : 'dark-content'}
        // backgroundColor={backgroundStyle.backgroundColor}
        />

        <MainStack />
      </SafeAreaView>
    </NavigationContainer>
  );
}

export default App;
