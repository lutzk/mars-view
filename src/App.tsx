import React from 'react';
import { MainStack } from '@navigation/MainStack';
import AppProviders from './AppProviders';

const App = () => {
  return (
    <AppProviders>
      <MainStack />
    </AppProviders>
  );
};

export default App;
