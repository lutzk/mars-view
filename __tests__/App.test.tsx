import 'react-native';
import React from 'react';
import App from '../src/App';

// Note: import explicitly to use the types shiped with jest.
import { it, expect } from '@jest/globals';

// Note: test renderer must be required after react-native.
import { render } from '@testing-library/react-native';

/**
 * https://github.com/react-navigation/react-navigation/issues/10943
 */

it('renders correctly', () => {
  const app = render(<App />).toJSON();
  expect(app).toMatchSnapshot();
});
