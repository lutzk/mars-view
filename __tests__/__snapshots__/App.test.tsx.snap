// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
    ___load persist data____
  </Text>
</View>
`;
